sdk/storage/inc/adls/protocol/adls_rest_client.hpp

```diff
@@ -0,0 +1,2220 @@
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <functional>
+#include <iostream>
+#include <map>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+#include "http/curl/curl.hpp"
+#include "http/http.hpp"
+#include "json.hpp"
+
+namespace Azure
+{
+namespace Storage
+{
+namespace Datalake
+{
```

Comment:
```suggestion
namespace Azure { namespace Storage { namespace Datalake {
```

Reply:
Cannot find my previous comment, I think the clang format correctly handled this.